app/controlplane/api/controlplane/v1/workflow_run.proto

```diff
@@ -91,6 +100,9 @@ message AttestationServiceCancelRequest {
     not_in: [0]
   }];
   string reason = 3;
+  // This parameter is not needed by Robot Account since they have the workflowID embedded.
+  // API Tokens will send the parameter explicitly
+  string workflow_name = 4;
```

Comment:
I am struggling to understand why we need workflow_name in this situation. If you have the `run` and the organization in the token, aren't you supposed to also be able to know the workflow and make sure you have access to it?